Classes/common/CDTIncrementalStore.h

```diff
@@ -126,7 +126,7 @@ typedef NS_ENUM(NSInteger, CDTIncrementalStoreErrors) {
     CDTISErrorMetaDataMismatch,
     CDTISErrorNoRemoteDB,
     CDTISErrorSyncBusy,
-    CDTISErrorNotSupported
+    CDTISErrorNotSupported,
```

Comment:
NIT, get rid of this `,` and this hunk goes away.
I always found it odd the objc allowed this `,` in the last position when C/C++ didn't (At least not in C99 :smile:)